### DISCUSSION TITLE
天门山 | /posts/tianmen-mountain/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#650] [Announcements]
https://yuanfan.rbind.io/posts/tianmen-mountain/

**Comment by hongtaoh** [comment]
也不知道为啥张家界在韩国那么有名。不过我听说因为免签，现在上海也是挤满了韩国游客。

**Reply by earfanfan** [reply]
>也不知道为啥张家界在韩国那么有名。

这点我也非常好奇，路上领队给大家讲了一种说法，大致是张家界这边地貌奇特，山里又很多洞，而韩国人认为其中一个洞里是他们的起源，加上韩国那边也重视孝道，如果感恩父母就把父母送到张家界来玩。

不过我自己又查了一下，原来张家界30年前就修建了机场，开通了与韩国的航线，多年旅游业的发展积累可能才有今天的声名和效果。

回想起来那边的山里确实有些人类无法到达的洞，我去的那天早上山上的温度在零度以下，爬山的时候还远远看到了冰挂。

**Comment by tianheg** [comment]
你拍的风景照真美呀！同行人照片里的你很可爱哟 :smiley: ！

**Reply by earfanfan** [reply]
嗷一声，你的赞美我收到啦，我很受用哟，谢谢你。
